src/main/java/com/ericsson/ei/handlers/EventHandler.java

```diff
@@ -33,7 +33,7 @@
 @Component
 public class EventHandler {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(EventHandler.class);
+    private static Logger log = LoggerFactory.getLogger(EventHandler.class);
```

Comment:
Why is the logger not final?

Reply:
Why was final remove and why was it renamed?